docs/subnets/reference-cli-commands.md

```diff
@@ -236,6 +242,43 @@ avalanche subnet import [subnetPath] [flags]
 
 <!-- markdownlint-enable MD013 -->
 
+#### Import from a public network 
```

Comment:
Title cases on heading

Reply:
I was surprised to see that titlecase.com suggests 
"Import from a Public Network" instead of what I would have instinctively applied: "Import From A Public Network". I went for the former.

Reply:
There are two outputs:
<img width="1277" alt="Screen Shot 2022-12-05 at 9 42 47 AM" src="https://user-images.githubusercontent.com/12171681/205693311-cbce23b0-cea8-4653-adbe-e3d7fb71a97a.png">